<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchProcessRawSessions</class>
 <widget class="QMainWindow" name="BatchProcessRawSessions">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>255</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Batch Process Raw Sessions</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../brl_logo.gif</normaloff>../brl_logo.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The top level directory containing all the sessions</string>
    </property>
    <property name="text">
     <string>Root Directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The first session to process</string>
    </property>
    <property name="text">
     <string>Start Session</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The last session to process</string>
    </property>
    <property name="text">
     <string>End Session</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtRootDir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>36</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The top level directory containing all the sessions</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbStartSess">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>67</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The first session to process</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbEndSess">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>98</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The last session to process</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRootSel">
    <property name="geometry">
     <rect>
      <x>493</x>
      <y>34</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Data settings</string>
    </property>
    <widget class="QPushButton" name="btnProcess">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>180</y>
       <width>70</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpBoxLSType">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Light Source Type</string>
     </property>
     <widget class="QRadioButton" name="rdoVIS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>VIS</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoNIR">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="text">
       <string>NIR</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoVISNIR">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>VISNIR</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="cbUseAdaptivePS">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use Adaptive light source vectors to generate results which take into account&lt;br/&gt;the pixel position in the field of view and adjust the light source vector accordingly.&lt;br/&gt;Select the location of the AdaptiveLSV.npz generated by the Generate Adaptive LSV tool&lt;br/&gt;in the box below&lt;br/&gt;If this is unchecked standard PS will be used.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Use Adaptive PS?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdaptiveLSVFileSel">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>150</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtAdaptiveLSVFile">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Select the location of the AdaptiveLSV.npz file</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Where to output the resultant masks</string>
     </property>
     <property name="text">
      <string>AdaptiveLSVs.npz location</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="grpBoxLSOverride">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This setting is only respected in Adaptive PS mode&lt;br/&gt;For example used when all 8 lights have been used to capture, but you only need  4 corresponding to NIR, you would select NIR.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>Override Light Source Type</string>
     </property>
     <widget class="QRadioButton" name="rdoOverrideVIS">
      <property name="geometry">
       <rect>
        <x>69</x>
        <y>20</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>VIS</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoOverrideNIR">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>109</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="text">
       <string>NIR</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoOverrideVISNIR">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>VISNIR</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdoOverrideNone">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>edtRootDir</zorder>
   <zorder>cmbStartSess</zorder>
   <zorder>cmbEndSess</zorder>
   <zorder>btnRootSel</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>edtRootDir</tabstop>
  <tabstop>btnRootSel</tabstop>
  <tabstop>cmbStartSess</tabstop>
  <tabstop>cmbEndSess</tabstop>
  <tabstop>edtAdaptiveLSVFile</tabstop>
  <tabstop>btnAdaptiveLSVFileSel</tabstop>
  <tabstop>btnProcess</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
